content/docs/nav.yml

```diff
@@ -11,7 +11,6 @@
     - id: release-channels
       title: Release Channels
 - title: Main Concepts
-  isOrdered: true
```

Comment:
This should be intentional since the documentation is sequential and refers to older pages in the next page.